### PR TITLE
update github labels

### PR DESCRIPTION
**Pull request by pdabelf5**
### Proposed changes

Add labels for `go`, `python`, `docker` & `github_actions`

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by vepatel — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -11,7 +11,19 @@ chore:
...
+  - changed-files:
+    - any-glob-to-any-file: ['.github/**/*.yml', '.github/**/*.yaml', '.github/**/*.md']
+
+go:
+  - changed-files:
+    - any-glob-to-any-file: ['**/*.go', 'go.mod', 'go.sum']
+
+python:
+  - changed-files:
+    - any-glob-to-any-file: ['**/*.py', '**/requirements.txt']
```
Comment (vepatel) [inline .github/labeler.yml:26]: tests use some yaml files and secrets as well

Comment (vepatel) [inline .github/labeler.yml:26]:
```suggestion
    - any-glob-to-any-file: ['tests/**/*.py', 'tests/**/requirements.txt', 'tests/**/*.yaml']
```

Comment (pdabelf5) [reply .github/labeler.yml]: are they `python` changes or fall under the `test` label?

**Review by jjngx — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (vepatel) [reply .github/labeler.yml]: technically both, are we excluding tests from python?

**Review by shaun-nx — APPROVED** [review]

**Pull request merged**